database/index.js

```diff
@@ -0,0 +1,63 @@
+const { Client } = require('pg');
+const Promise = require('bluebird');
+
+const client = new Client({
+  user: process.env.LENA_ORDERS_DB_USER || 'lena',
+  host: process.env.LENA_ORDERS_DB_HOST || 'localhost',
+  database: process.env.LENA_ORDERS_DB_data || 'transactions',
+  port: process.env.LENA_ORDERS_DB_PORT || 5432,
+});
+
+client.connect();
+
+const createOrder = (userId, shippingAddress) => {
+  const queryStr = `
+    INSERT INTO orders
+    (user_id, shipping_address, delivery_status, created_at)
+    VALUES
+    ($1, $2, 'created', $3)
+  `;
+  const currDate = new Date().toISOString().slice(0, 10);
```

Comment:
Not a good idea to hardcode the slice index - even thought ISOStrings have consistent formats. Can either use a library to format it or use native getYear, getMonth, getDate functions.

Reply:
Good idea, I found the npm package `yyyy-mm-dd`